packages/prisma/schema.prisma

```diff
@@ -201,6 +201,10 @@ model EventType {
   secondaryEmailId Int?
   secondaryEmail   SecondaryEmail? @relation(fields: [secondaryEmailId], references: [id], onDelete: Cascade)
 
+  restrictionScheduleId Int?
+  useBookerTimezone     Boolean   @default(false)
+  restrictionSchedule   Schedule? @relation("restrictionSchedule", fields: [restrictionScheduleId], references: [id])
```

Comment:
I would order new fields so that `restrictionSchedule` and `restrictionScheduleId` are next to each other
```
restrictionScheduleId
restrictionSchedule
useBookerTimezone
```